src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverterSettings.cs

```diff
@@ -0,0 +1,250 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+
+namespace System.Text.Json.Serialization
+{
+    public class JsonConverterSettings
+    {
+        private volatile JsonMemberBasedClassMaterializer _classMaterializerStrategy;
+        private JsonClassMaterializer _classMaterializer;
+        private int _defaultBufferSize = 16 * 1024;
```

Comment:
Create a named constant for the initial default buffer size?

Reply:
Done